apps/web/components/DisputeButton.tsx

```diff
@@ -231,8 +232,8 @@ export const DisputeButton: FC<Props> = ({ proposalData }) => {
   };
 
   const content = (
-    <div className="flex flex-col gap-10 lg:min-w-[800px]">
-      {isDisputed || proposalStatus === "rejected" ?
```

Comment:
I'd like to see why the proposal is rejected, so we need to display the dispute button for this